### DISCUSSION TITLE
Jan has Assignable Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by dan-menlo** [janhq/jan#3572] [Feature Ideas]
## Overview

- We should have Keyboard shortcuts that can be re-assigned

**Comment by ux-han** [comment]
![Screenshot 2024-10-02 at 8 39 55 AM](https://github.com/user-attachments/assets/31a73a26-2afd-4906-a12e-7ffadf851736)

**Comment by ux-han** [comment]
Related: 
https://github.com/janhq/jan/issues/2757
https://github.com/janhq/jan/issues/3587
https://github.com/janhq/jan/issues/3767

**Comment by jachristian11** [comment]
I would like to request a keyboard shortcut to set the focus on the 'Ask me anything' input field. Currently, I find it convenient that browsers use Ctrl+L and/or Alt+D to focus the search/address bar. I would appreciate it if a similar shortcut could be implemented in the application.

After copying text from the window and pasting it into another application, followed by Alt+Tabbing back to the Jan application, the focus will be on the highlighted text. It can then take multiple tab presses to reach the text entry field if the body text had any hyperlinks. Having a dedicated keyboard shortcut would ensure consistent access to the text input location.
